pallets/delegation/src/lib.rs

```diff
@@ -374,11 +389,15 @@ pub mod pallet {
 				Error::<T>::DelegationNotFound
 			);
 
+			let max_parent_checks = max_parent_checks.min(T::MaxParentChecks::get());
+
 			ensure!(
 				Self::is_delegating(&invoker, &delegation_id, max_parent_checks)?,
 				Error::<T>::UnauthorizedRevocation
 			);
 
+			let max_revocations = max_revocations.min(T::MaxRevocations::get());
```

Comment:
and here

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/189/commits/0ddb538a40f37949049b516c3fa96c4f938e533b